mentors/javascript/m.json

```diff
@@ -54,5 +54,13 @@
     "link_url": "https://mattmccarley.com/",
     "avatar_url": "https://avatars2.githubusercontent.com/u/16927488?s=400&u=87d5923bf84a4fe44b1af70f8fb731927dd8b248&v=4",
     "bio": "Curriculum Developer <- Wannabe CS Nerd <- Front-End Developer & UX/UI Designer <- Teacher"
+  },
+  {
+    "github_username": "marco-quintella",
+    "name": "Marco Quintella",
+    "link_text": "Github Profile",
```

Comment:
All looks fine with exception of Github needs to be properly capitalized.  GitHub

Reply:
It's true. My mistake! Corrected.